### PR TITLE
adding illustrative example of proposed callback

### PR DESCRIPTION
**Pull request by philipmac**
first attempt of proposal

### CODE REVIEW

File: docs/source/demo_callback.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                        {
+                            "type": "keyImage",
+                            "path": "LNII/sbest/sbest-2023-0526-Rename/TEM-Tomo-2023-0526-Rename1/LNII/F13-TS005/F13-TS005_a.jpg"
+                        },
+                        {
+                            "type": "neuroglancerZarr",
+                            "path": "LNII/sbest/sbest-2023-0526-Rename/TEM-Tomo-2023-0526-Rename1/LNII/F13-TS005/F13-TS005_a.zarr",
+                            "metadata": {
+                                "neuroglancerPrecomputedMin": "-55",
+                                "neuroglancerPrecomputedMax": "60",
```
Comment (blowekamp) [inline docs/source/demo_callback.json:22]: This should be omitted as it is documented in another file.

File: docs/source/demo_callback.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+                                "neuroglancerPrecomputedFloor": "-638",
+                                "neuroglancerPrecomputedLimit": "460"
+                            }
+                        },
+                        {
```
Comment (blowekamp) [inline docs/source/demo_callback.json:27]: These additional assets are not going to be used use CZI, and can be removed.

File: docs/source/demo_callback.json [hunk 3]
```diff
@@ -0,0 +1,90 @@
+{
+    "files": [
+        {
+            "primaryFilePath": "LNII/sbest/sbest-2023-0526-Rename/TEM-Tomo-2023-0526-Rename1/LNII/F13-TS005.czi",
```
Comment (blowekamp) [inline docs/source/demo_callback.json:4]: Perhaps we could have a more spacialopics like path?
"TEM-Tomo-2023-0526-Rename1" -> "SpacialOpimix_example-2023-0712"

**Review by mbopfNIH — COMMENTED** [review]
I'm trying to follow the spatial omics stuff, at least at a high level. Could I listen in on you meetings? It would help me in the future even if I don't contribute now.

**Comment by philipmac** [comment]
> I'm trying to follow the spatial omics stuff, at least at a high level. Could I listen in on you meetings? It would help me in the future even if I don't contribute now.

yeah, I'll include you in the next meeting where we try to all agree on this stuff.
There's two problems that we're looking at ATM, 
1) the "metadata" that's required for the neuroglancer viewer needs extending. 
2) some input files, (`czi` for example) can have >1 image within them. This leads to a change in the schema. At the moment we;re writing down concrete examples to ensure we're clear about what is on our minds.

File: docs/source/demo_callback.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
+{
+  "files": [
+    {
+      "primaryFilePath": "LNII/sbest/sbest-2023-0526-Rename/Spatial_example-2023-0526-Rename1/LNII/F13-TS005.czi",
+      "fileMetadata": {
+        "TBD": "TBD"
+      },
+      "imageRecords": [
```
Comment (philipmac) [inline docs/source/demo_callback.json:8]: maybe s/imageRecords/imageSets/g ?

File: docs/source/demo_callback.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
+{
+  "files": [
+    {
+      "primaryFilePath": "LNII/sbest/sbest-2023-0526-Rename/Spatial_example-2023-0526-Rename1/LNII/F13-TS005.czi",
+      "fileMetadata": {
+        "TBD": "TBD"
```
Comment (philipmac) [inline docs/source/demo_callback.json:6]: the data is stored as XML, but we use JSON to store here.

File: docs/source/demo_callback.json (hunk 4 above)
Comment (mbopfNIH) [reply docs/source/demo_callback.json:8]: A bit pedantic, but isn't "imageSet" already plural? Unless we have multiple sets of images, I think it should be imageSet.

**Pull request merged**